### PR TITLE
Update installation docs

### PR DESCRIPTION
**Pull request by roomrys**
### Description
Update the installation docs.

### Types of changes

- [ ] Bugfix
- [ ] New feature
- [ ] Refactor / Code style update (no logical changes)
- [ ] Build / CI changes
- [x] Documentation Update
- [ ] Other (explain)

### Does this address any currently open issues?
[list open issues here]

### Outside contributors checklist

- [ ] Review the [guidelines for contributing](https://github.com/talmolab/sleap/blob/develop/docs/CONTRIBUTING.md) to this repository
- [ ] Read and sign the [CLA](https://github.com/talmolab/sleap/blob/develop/sleap-cla.pdf) and add yourself to the [authors list](https://github.com/talmolab/sleap/blob/develop/AUTHORS)
- [ ] Make sure you are making a pull request against the **develop** branch (not *main*). Also you should start *your branch* off *develop*
- [ ] Add tests that prove your fix is effective or that your feature works
- [ ] Add necessary documentation (if appropriate)

#### Thank you for contributing to SLEAP!
:heart:

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Installation
 
-SLEAP can be installed as a Python package on Windows, Linux and Mac OS X. We currently provide {ref}`experimental support for Apple Silicon Macs <apple-silicon>`.
+SLEAP can be installed as a Python package on Windows, Linux, Mac OS X, and Mac OS Apple Silicon. We currently provide {ref}`experimental support for Apple Silicon Macs <apple-silicon>`.
```
Comment (talmo) [inline docs/installation.md:3]:
```suggestion
SLEAP can be installed as a Python package on Windows, Linux, Mac OS X, and Mac OS Apple Silicon.`.
```

No longer experimental 🎉

File: docs/installation.md [hunk 2]
```diff
@@ -12,62 +12,108 @@ local:
...
+
+```bash
+mamba create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.1
+```
+
+**Mac OS X** and **Apple Silicon**
+
 ```bash
-conda create -y -n sleap -c sleap -c nvidia -c conda-forge sleap=1.3.1
+mamba create -y -n sleap -c conda-forge -c anaconda -c sleap sleap=1.3.1
```
Comment (talmo) [inline docs/installation.md:113]: Will this command not work on Macs if you include `-c nvidia`?

**Review by talmo — APPROVED** [review]
Minor suggestions but otherwise lgtm

File: docs/installation.md (hunk 2 above)
Comment (roomrys) [reply docs/installation.md:113]: Haven't tried it, correct me if I am wrong, but I believe we don't need any packages from nvidia for macs. Were you looking to standardize the installation command? I was aiming to minimize to just what we needed.

**Pull request merged**